providers/class-two-factor-fido-u2f-admin.php

```diff
@@ -309,10 +320,10 @@ public static function delete_link( $item ) {
 	public static function wp_ajax_inline_save() {
 		check_ajax_referer( 'keyinlineeditnonce', '_inline_edit' );
 
-		require( TWO_FACTOR_DIR . 'providers/class.two-factor-fido-u2f-admin-list-table.php' );
+		require TWO_FACTOR_DIR . 'providers/class-two-factor-fido-u2f-admin-list-table.php';
 		$wp_list_table = new Two_Factor_FIDO_U2F_Admin_List_Table();
 
-		if ( ! isset( $_POST['keyHandle'] ) ) {
+		if ( ! isset( $_POST['key_handle'] ) ) {
```

Comment:
This too would need to be reverted if we're keeping the vendor libraries away from WPCS.

Reply:
Fixed